integration-tests/browser_integration_test.go

```diff
@@ -65,8 +67,18 @@ func TestBrowserIntegration(t *testing.T) {
 
 	// standaloneOrder is an order that will be sent to the network by the
 	// standalone node.
-	ethClient := ethclient.NewClient(ethRPCClient)
-	standaloneOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	standaloneOrder := scenario.NewSignedTestOrder(t, orderopts.SetupMakerState(true))
+
+	// We also need to set up the maker state for the order that will be created in the browser (we don't care
+	// if this order exactly matches the one created in the browser, we just care about makerAddress,
+	// makerAssetData, and makerAssetAmount).
+	scenario.NewSignedTestOrder(t,
+		orderopts.SetupMakerState(true),
+		orderopts.MakerAddress(constants.GanacheAccount1),
+		orderopts.MakerAssetData(scenario.ZRXAssetData),
+		orderopts.MakerAssetAmount(big.NewInt(1000)),
```

Comment:
I think it's better to not implicitly depend on the default values as it can lead to confusing breakages if something changes. We are explicitly stating that the order created in the browser will have a certain  `MakerAddress`, `MakerAssetData`, and `MakerAssetAmount`, and I already included a comment in the code about this:

```
	// We also need to set up the maker state for the order that will be created in the browser (we don't care
	// if this order exactly matches the one created in the browser, we just care about makerAddress,
	// makerAssetData, and makerAssetAmount).
```

Should I reword or change the comment in some way to make it more clear?

Reply:
No, that comment is clear enough. I meant that we could add a comment about the reliance on default values in the event that you implemented that. I see your point that it is a code smell though.